Assets/Scripts/UI/Core/UISFXHandler.cs

```diff
@@ -54,4 +54,12 @@ public void ToggleActive(bool isActive)
         this.isActive = isActive;
     }
 
+    public void ToggleSoundsEnabled(bool areEnabled)
```

Comment:
This is a little misleading to call it `Toggle` if the function does nothing is `areEnabled` is set to `true`. I think you should change this in one of two ways:
1. Remove the `bool` arg and rename it to something like: `DisableSounds`
2. Remember the original values of these click sounds and If `areEnabled` is  set to true, restore them accordingly